### PR TITLE
Add a code block fence for example-ingress.yaml on tasks/access-application-cluster/ingress-minikube

### PR DESCRIPTION
**Pull request by shuuji3**
**What is the problem**
Currently, the example YAML code on [tasks/access-application-cluster/ingress-minikube](https://kubernetes.io/docs/tasks/access-application-cluster/ingress-minikube/#create-an-ingress-resource) has extra indentations and does not have a code fence. This appends an extra space for each line. Also, without a code fence, the code is not highlighted properly.

**What this PR fixes**
- Add a `yaml` code fence and remove extra indentation to amend the above problem.
- Also, this PR clean up unnecessary spaces from the page.

### CODE REVIEW

File: content/en/docs/tasks/access-application-cluster/ingress-minikube.md [hunk 1]
```diff
@@ -132,37 +132,39 @@ The following file is an Ingress resource that sends traffic to your Service via
...
+    spec:
+      rules:
+      - host: hello-world.info
+        http:
+          paths:
+          - path: /
+        backend:
+          serviceName: web
+          servicePort: 8080
+    ```
```
Comment (shuuji3) [inline content/en/docs/tasks/access-application-cluster/ingress-minikube.md:151]: Here is an added code fence.

**Comment by shuuji3** [comment]
Page preview: https://deploy-preview-22984--kubernetes-io-master-staging.netlify.app/docs/tasks/access-application-cluster/ingress-minikube/#create-an-ingress-resource

/assign @savitharaghunathan

**Comment by kbhawkey** [comment]
Thanks @shuuji3 .
/approve

**Comment by shuuji3** [comment]
@yakirch Thank you for pointing out it. Sorry, this is my fault. :pray: #23047 will fixes this issue.

**Pull request merged**